### PR TITLE
Remove array-swap! from 179.sld

### PR DESCRIPTION
**Pull request by gambiteer**
179.sld:

1.  Remove array-swap!

compare-code-and-srfi.scm:

1.  Add code to compare srfi and sld file.

srfi-179.scm:

1.  Add new line to errata

srfi-179.html:

1.  Regenerate from srfi-179.scm

### CODE REVIEW

**Comment by arthurgleckler** [comment]
There's no need to document changes outside the document itself in the Status section.  Please remove the change there.

Thanks.

**Comment by gambiteer** [comment]
Aha, so people are supposed to look at the github history to learn whether there have been any changes in the other files after finalization?

**Comment by arthurgleckler** [comment]
On Sun, Oct 11, 2020 at 3:33 PM Bradley Lucier <notifications@github.com>
wrote:

> Aha, so people are supposed to look at the github history to learn whether
> there have been any changes in the other files after finalization?
>
I see your point. The reason is this: the document itself is the
"normative" part of the standard.  In other words, it — and only it —
defines what the SRFI means.  The implementation is only a sample
implementation, which means that if there is a disagreement between it and
the document, the document wins.  The erratum notice in the Status section
is there to explain changes made to the normative part of the standard —
the document.  It's meant to be brief and to the point about just that.
Many SRFIs have many changes made to their sample implementations after
finalization, and including all of those would make it hard to find changes
to the meaning of the SRFI, which are the most important changes.
Basically, the erratum notices in the Status section are a way for someone
who reads the document and thinks, "wait, I thought it said X, but now it
says Y.  What happened?" to get a quick answer.

But it's perfectly fine to announce a change to the sample implementation
on the mailing list.  That will reach everyone who is actively monitoring
the SRFI.

**Comment by arthurgleckler** [comment]
Thanks.  Merged and pushed.  I fixed a typo in the punctuation, so make sure to grab a copy and update your version.

**Comment by gambiteer** [comment]
OK, thanks.  Will you send something to the mail list?  (I ask because it always looks more official coming from you ;-)
